towhee/hub/bin/hub_tools.py

```diff
@@ -14,73 +14,97 @@
 
 import argparse
 from getpass import getpass
+from pathlib import Path
+from shutil import rmtree
 
 from towhee.hub.operator_manager import OperatorManager
 from towhee.hub.pipeline_manager import PipelineManager
 from towhee.hub.repo_manager import RepoManager
+from towhee.utils.git_utils import GitUtils
 
 
-def get_repo_manager(args):
-    if args.command in ['download', 'clone']:
-        manager = RepoManager(args.author, args.repo)
-    elif args.command == 'generate-yaml' or args.type == 'pyoperator':
-        manager = OperatorManager(args.author, args.repo)
-    elif args.type == 'nnoperator':
-        manager = OperatorManager(args.author, args.repo)
-    elif args.type == 'pipeline':
-        manager = PipelineManager(args.author, args.repo)
-    return manager
+def get_repo_obj(args):
+    if args.command == 'download':
+        repo_obj = RepoManager(args.author, args.repo)
+    elif args.command == 'clone':
+        repo_obj = GitUtils(args.author, args.repo)
```

Comment:
Why return `GitUtils`, and others are `Manger`.

Reply:
There are `create/init/generate-yaml/download/clone` methonds, and `clone` is for GitUtils, others for Manager.